### DISCUSSION TITLE
Ender 3V2 Neo Firmware

### DISCUSSION THREAD

**Discussion by deck55** [CrealityOfficial/Creality_Sonic_Pad#19] [General]
I would like to request to release of the Ender 3V2 Neo firmware.

**Comment by kamoteshake** [comment]
Seems like people already have asked for the firmware for 3v2 Neo. It's just Creality haven't been acknowledging these [requests](https://github.com/CrealityOfficial/Ender-3V2/issues/1). @CrealityTech could you at least give the people some updates? this was requested Oct of 2022.

**Comment by Lash-L** [comment]
@CrealityTech You said it would be coming soon in September, do you have any update for us?

**Comment by Toby0501** [comment]
@CrealityTech Any update on this?

**Comment by Lukeshystalker** [comment]
Any way possible that you all can work on a firmware for the max neo that fix host actions in octoprint?

**Comment by Natgho** [comment]
Update here...
